include/utils/BaselineCorrectionUtils.h

```diff
@@ -0,0 +1,71 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     MASTODON                  */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+// This code was implemented in collaboration with Christopher J. Wong
+// (chris.wong@utah.edu) from the University of Utah.
+
+#pragma once
+
+// LIBMESH includes
+#include "DenseMatrix.h"
+#include "libmesh/dense_vector.h"
+
+/**
+ * This namespace contains the functions used for the calculations corresponding
+ * to the time history adjustment procedure in BaselineCorrection
+ **/
+namespace BaselineCorrectionUtils
+{
+/// Evaluates an integral over a single time step with Newmark-beta method
+/// Also is used as simple trapezoidal rule when gamma = 0.5.
+Real newmarkGammaIntegrate(const Real & u_ddot_old,
+                           const Real & u_ddot,
+                           const Real & u_dot_old,
+                           const Real & gamma,
+                           const Real & dt);
+
+/// Evaluates a double integral over a single time step with Newmark-beta method
+Real newmarkBetaIntegrate(const Real & u_ddot_old,
+                          const Real & u_ddot,
+                          const Real & u_dot_old,
+                          const Real & u_old,
+                          const Real & beta,
+                          const Real & dt);
+
+/// Solves linear normal equation for minimum acceleration square error
```

Comment:
Is there a simpler explanation to the phrase `linear normal equation`?

Reply:
@somu15 This is what it is referred to as in terms of linear algebra jargon. For example, see [section 8.2 (page 510) of this textbook](https://fac.ksu.edu.sa/sites/default/files/numerical_analysis_9th.pdf), or see this [Wikipedia page](https://en.wikipedia.org/wiki/Polynomial_least_squares). 

It's just the equation that solves the least squares polynomial. I suppose I could just straight-up call it this instead.